config/ks-core/templates/kubesphere-config.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if .Values.config.create -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kubesphere-config
+  namespace: kubesphere-system
+data:
+  kubesphere.yaml: |
+    authentication:
+      authenticateRateLimiterMaxTries: {{ .Values.config.authentication.authenticateRateLimiterMaxTries | default 10 }}
+      authenticateRateLimiterDuration: {{ .Values.config.authentication.authenticationRateLimiterDuration | default "10m0s" }}
+      loginHistoryRetentionPeriod: {{ .Values.config.authentication.loginHistoryRetentionPeriod | default "168h"  }}
+      maximumClockSkew: {{ .Values.config.authentication.maximumClockSkew | default "10s" }}
+      multipleLogin: {{ .Values.console.enableMultiLogin | default true }}
+      kubectlImage: {{ .Values.image.ks_kubectl_repo }}:{{ .Values.image.ks_kubectl_tag | default "latest" }}
+      jwtSecret: "{{ .Values.jwtSecret }}"
```

Comment:
If `jwtSecret` is empty use a random string instead.

Reply:
@wansir  How about using `default (sha1sum uuidv4 ) ` which will generate a hash of the random uuid?

Reply:
> @wansir How about using `default (sha1sum uuidv4 ) ` which will generate a hash of the random uuid?

[A key of the same size as the hash output (for instance, 256 bits for"HS256") or larger MUST be used with this algorithm.](https://tools.ietf.org/html/rfc7518#section-3.2)

```
jwtSecret:  {{ randAlphaNum 32 | quote }} `
```

Reply:
Resoved